src/config/config_manager.cc

```diff
@@ -1007,14 +1007,14 @@ void ConfigManager::addOption(config_option_t option, std::shared_ptr<ConfigOpti
 
 void ConfigManager::load(const fs::path& userHome)
 {
-    std::string temp;
+    fs::path temp;
     pugi::xml_node tmpEl;
 
     std::map<std::string, std::string> args;
     auto self = getSelf();
     std::shared_ptr<ConfigSetup> co;
 
-    log_info("Loading configuration from: {}", filename.c_str());
+    log_info("Loading configuration from: {}", filename.string().c_str());
     pugi::xml_parse_result result = xmlDoc->load_file(filename.c_str());
```

Comment:
`fs::path` has a `c_str()` method https://en.cppreference.com/w/cpp/filesystem/path/native. No need to do two steps

Reply:
Not quite. c_str() can be a wchar_t.